static/admin/css/base.css

```diff
@@ -0,0 +1,850 @@
+/*
```

Comment:
To anyone else reading this: These huge files that seem unrelated to the actual change should have been included in the initial project commit just because we're going to need them when we're on a production server. We included them now because we can.